### DISCUSSION TITLE
I can't get past the Title screen.

### DISCUSSION THREAD

**Discussion by Cottoncandyflame** [smplayer-dev/smplayer#27] [Q&A]
Every time I try to play a DVD, it will play the advertisements then a to the movies title screen. It will not play the movie itself. The Title section under browse says empty. I can get it to play by switching from mpv to mplayer in preferences, but the problem with that is, mplayer doesn't let me go backwards one frame at a time and being able to study movies frame by frame is the only reason I picked this software. 
Also, for the past few hours, every time I try to pause it briefly minimizes the window before flashing back up a half second later. I must have changed something in the process of trying to fix the earlier problem but I  don't know exactly what I did.
Please help

**Comment by smplayer-dev** [comment]
> Every time I try to play a DVD, it will play the advertisements then a to the movies title screen. It will not play the movie itself. The Title section under browse says empty. I can get it to play by switching from mpv to mplayer in preferences, but the problem with that is, mplayer doesn't let me go backwards one frame at a time and being able to study movies frame by frame is the only reason I picked this software.

I think you should report this in the [issues](https://github.com/smplayer-dev/smplayer/issues) section. Don't forget to send the logs.

> Also, for the past few hours, every time I try to pause it briefly minimizes the window before flashing back up a half second later. I must have changed something in the process of trying to fix the earlier problem but I don't know exactly what I did.
> Please help

Maybe you enabled the option to stay on top during playback (menu Video).
